index.js

```diff
@@ -256,6 +263,20 @@ var handleCloudWatch = function(event, context) {
   } else if (message.NewStateValue === "OK") {
       color = "good";
   }
+  
```

Comment:
It would be good to default `accountName` to the `accountId` to cover the case where the account isn't in the mapping. Not a hard requirement but makes this less fragile.